tfe/resource_tfe_organization.go

```diff
@@ -166,6 +173,11 @@ func resourceTFEOrganizationUpdate(d *schema.ResourceData, meta interface{}) err
 		options.SendPassingStatusesForUntriggeredSpeculativePlans = tfe.Bool(sendPassingStatusesForUntriggeredSpeculativePlans.(bool))
 	}
 
+	// If cost_estimation_enabled is supplied, set it using the options struct.
+	if assessmentsEnforced, ok := d.GetOkExists("assessments_enforced"); ok {
+		options.assessmentsEnforced = tfe.Bool(assessmentsEnforced.(bool))
```

Comment:
I think this should be:
```suggestion
		options.AssessmentsEnforced = tfe.Bool(assessmentsEnforced.(bool))
```